prow/cluster/pushgateway_deployment.yaml

```diff
@@ -44,7 +44,7 @@ data:
     pid /run/nginx.pid;
     error_log /dev/stdout;
     events {
-      worker_connections 20;
+      worker_connections 200;
```

Comment:
FWIW this defaults to 512 and 1024 is pretty common (due to ulimits) http://nginx.org/en/docs/ngx_core_module.html#worker_connections

Reply:
The increase seemed to fix the problem and we use 1024 for both velodrome and the submit-queue, so I'm going to bump this to 1024 as well so that we don't hit the limit again.

Reply:
https://github.com/kubernetes/test-infra/pull/8501